alwaysontop/render/index.js

```diff
@@ -37,11 +37,14 @@ class AlwaysOnTop extends EventEmitter {
      * Creates new instance.
      *
      * @param {JitsiIFrameApi} api - the Jitsi Meet iframe api object.
+     * @param {boolean} showOnPrejoin - whether AOT should show on prejoin screen.
      */
-    constructor(api) {
+    constructor(api, showOnPrejoin) {
```

Comment:
Can you please add an `options` object instead, so we can add more things if / when we need to?